kde/kmymoney4-devel/Portfile

```diff
@@ -7,11 +7,11 @@ PortGroup           kde4    1.1
 
 fetch.type          git
 git.url             git://anongit.kde.org/kmymoney
-git.branch          485aea8e20604cf79612f62d1393fde9872c0273
+git.branch          ebeeced0715a82327ae0e2ee833cf93fc3029366
 
 name                kmymoney4-devel
-version             4.8-20161022
-revision            2
+version             4.8.1
+revision            1
```

Comment:
The `revision` line should be deleted because it starts at 0 when increasing the version and that is the default value.